File: array/832.py
class Solution:
    def flipAndInvertImage(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        for arr in A:
            arr.reverse()
            arr[::] = map(lambda a : a ^ 1, arr)
        return A
        # return [[i^1 for i in arr[::-1]] for arr in A]

File: array/832.cc
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& A) {
        for (auto& row : A) {
            std::reverse(row.begin(), row.end());
            for (auto& i: row) {
                i ^= 1;
            }
        }
        return A;
    }
};
